src/main/java/frc/robot/subsystems/climber/Climber.java

```diff
@@ -81,6 +85,8 @@ private Climber() {
             armTower.setColor(new Color8Bit(Color.kBlue));
         }
 
+        mainMotor.configSelectedFeedbackSensor(FeedbackDevice.RemoteSensor0, 1, Constants.TALON_TIMEOUT);
```

Comment:
The CTRE docs say ([source](https://docs.ctre-phoenix.com/en/stable/ch14a_BringUpRemoteSensors.html#bring-up-remote-sensors)):
> The remote sensor filter feature of the Talon SRX/Victor SPX allows for the processing of sensor values provided from other remote CAN bus devices. In other words, a Talon SRX or Victor SPX can execute closed-loop modes with sensor values **sourced by other Talons, CANifiers, or Pigeon IMUs on the same CAN-bus**. (emphasis mine)

AFAIK you plan to use REV Robotics' Through Bore Encoder which doesn't have a CAN-bus connector. Assuming you don't plan to use a CANifier, how are you going to use the encoder as a remote sensor?

Reply:
You might be right we will check with the captain how the remote sensor is connected